Service.Core/Network/Public/Include/Network/Connection.hpp

```diff
@@ -332,12 +332,12 @@ class Connection : public std::enable_shared_from_this<Connection>
     }
 
     /**
-     * @brief Method for connection to server from client side.
-     * @details Only clients can connect to servers and make a request asio attempts \
-     * to connect to an endpoint.
+     * @brief Method for connection to service from client side.
```

Comment:
The method that initiates the connection of clients to the service. At attempt to call it on `backend` side do nothing